client/my-sites/patterns/components/pattern-preview/style.scss

```diff
@@ -22,6 +22,14 @@
 	justify-content: space-between;
 	position: relative;
 
+	iframe {
+		pointer-events: initial !important;
+	}
+
+	> div[aria-hidden="true"] {
+		pointer-events: initial !important;
+	}
```

Comment:
Why do we need this?

Reply:
I thought this was needed, but it looks like you're right – it's not. I'll remove it 👍